docs/Makefile

```diff
@@ -7,13 +7,14 @@ SPHINXBUILD   = sphinx-build
 SPHINXPROJ    = PyTorch
 SOURCEDIR     = source
 BUILDDIR      = build
+PYCMD         = python
 
 # Put it first so that "make" without argument is like "make help".
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
 figures:
-	python3 source/scripts/build_activation_images.py
+	@$(PYCMD) source/scripts/build_activation_images.py
```

Comment:
I have made `PYCMD` a Makefile argument.

Reply:
Why the `@` at the beginning of the line?

Reply:
I thought it is not required to print that command. If it is otherwise, I can change it.

Reply:
Right, I forgot it silences `make`. It's ok to leave it